contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol

```diff
@@ -42,10 +44,29 @@ contract FxMintableERC20ChildTunnel is Ownable, FxBaseChildTunnel, Create2 {
         // check if mapping is already there
         require(rootToChildToken[rootToken] == address(0x0), "FxMintableERC20ChildTunnel: ALREADY_MAPPED");
         rootToChildToken[rootToken] = childToken;
-
+        emit TokenMapped(rootToken,childToken);
+        
         // initialize child token with all parameters
         FxERC20(childToken).initialize(address(this), rootToken, name, symbol, decimals);
     }
+    
+    //To mint tokens on child chain
+    function mintToken(address childToken, uint256 amount) public {
```

Comment:
Shouldn't this be `ownerOnly` or something? Or you can make it abstract?

Reply:
Wasn't entirely sure of this but yeah I can make this a `virtual` function. Or it might be better to add a `ownerOnly` functionality